config/routes.rb

```diff
@@ -3,6 +3,8 @@
   devise_for :customers
   root "pages#index"
 
+  get "/customers/dashboard" => "customers#show"
```

Comment:
This is totally a matter of opinion, but if we're going to dress up the url anyways...why can't it just be `/dashboard` instead of `customers/dashboard`?